From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Wed, 28 Jan 2026 17:03:54 +0100
Subject: Exclude rust-vendor directory from workspace

---
 Cargo.toml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Cargo.toml b/Cargo.toml
index adbd0b5..b63ef2d 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,5 +1,6 @@
 [workspace]
 members = ["upki", "upki-mirror", "revoke-test", "rustls-upki"]
+exclude = ["rust-vendor"]
 resolver = "3"
 
 [workspace.package]
